Makefile

```diff
@@ -291,13 +291,8 @@ clean-docker: ## Deletes the docker containers for local development.
 govet: ## Runs govet against all packages.
 	@echo Running GOVET
 	$(GO) get -u golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow
-	$(GO) vet $(GOFLAGS) $(TE_PACKAGES) || exit 1
-	$(GO) vet -vettool=$(which shadow) $(GOFLAGS) $(TE_PACKAGES) || exit 1
-
-ifeq ($(BUILD_ENTERPRISE_READY),true)
-	$(GO) vet $(GOFLAGS) $(TE_PACKAGES) || exit 1
-	$(GO) vet -vettool=$(which shadow) $(GOFLAGS) $(EE_PACKAGES) || exit 1
-endif
+	$(GO) vet $(GOFLAGS) $(ALL_PACKAGES) || exit 1
```

Comment:
`$(ALL_PACKAGES)` is automatically set to include the enterprise packages if that source is present.